zds/utils/forms.py

```diff
@@ -143,3 +143,19 @@ def validate_no_empty_slug(self, tag):
     @property
     def errors(self):
         return self.__errors
+
+
+class FieldValidatorMixin(object):
```

Comment:
no need of `object` inheritance, we do not want to be compatible with py2

Reply:
Done, thanks.